src/documents/pages/fr/index.md

```diff
@@ -0,0 +1,59 @@
+descriptions de chaque sections du Guide et de ses fiches pratiques imprimables.
+
+<div class="idg-highlight-section" id="idg-index-dimensions-list">
+
+Au Centre de Recherche pour une Conception  Inclusive comme à l'Institut pour une Conception  Inclusive, nous mettons en
+avant les [**Trois Dimensions d'une Conception Inclusive**](http://idrc.ocadu.ca/about-the-idrc/49-resources/online-resources/articles-and-papers/443-whatisinclusivedesign)
+:
+
+* {% include "svg/icon-dimension.svg" %} **Prendre en compte la diversité et la singularité**
+
+  Quand les personnes sont définies par rapport à une moyenne, les besoins d'individus vivant des circonstances
+  particulières ou se situant à la marge paraissent encore plus variés. La plupart des personnes s'éloignent de la
+  moyenne dans l'expression de leurs besoins ou de leurs buts. Une unique solution ne peut donc pas être suffisamment
+  adaptée.
+
+* {% include "svg/icon-dimension.svg" %} **Utiliser Des outils et des processus favorisant l'inclusivité**
+
+  Les groupes de concepteurs devraient être aussi divers que possible en comprenant des individus qui partagent le même
+  vécu que les futurs utilisateurs. Cela suit le principe « rien n'est conçu pour nous s'il est conçu sans nous » en
+  évitant de reléguer une personne en situation de handicap au rôle de sujet d'une recherche ou à celui de participant
+  pour la forme à un exercice de conception.
+
+* {% include "svg/icon-dimension.svg" %} **Rechercher un impact et des bénéfices plus larges**
+
+  C'est la responsabilité du concepteur d'avoir connaissance du contexte d'utilisation et des retombées de ses
+  réalisations et de toujours rechercher un impact positif qui puisse être ressenti au delà du cercle des bénéficiaires
+  initialement prévus.
+
+</div>
+
+En respectant le cadre de ces trois dimensions dans notre pratique d'une conception inclusive, nous avons vécu plusieurs
+moments qui ont fait « tilt » et eu des opportunités d'apprentissage passionnantes. Notre but a été de capturer ces
+moments dans le Guide en un panorama de plus en plus large de [**Perspectives**](./perspectives). Nous avons également
+rassemblé une collection de [**Pratiques**](./pratiques), d'[**Outils**](./outils) et d'[**Activités**](./activites)
+dans le Guide qui donnent des exemples de mise en œuvre d'une conception inclusive.
+
+<div class="idg-highlight-section row">
+  <figure>
+  <figcaption class="small-12 medium-6 large-8 column">
+
+  Les sections et articles qui forment le Guide pour une Conception Inclusive ne suivent pas un ordre de lecture
+  pré-établi. Vous pouvez commencer où et quand bon vous semble pour vous, votre équipe et vos façons de travailler.
+  Reprendre le Guide à différentes étapes de votre projet peut être une bonne façon de l'intégrer à votre travail.
+
+  </figcaption>
+  <span class="small-12 medium-6 large-4 column"> {% include "svg/schematic.svg" %} </span>
```

Comment:
The aria label for this SVG uses the `<title>` and `<description>` elements from the SVG which needs to be translated.

I think the aria-label should be generated another way.